backend/functions/lib/adapters/utils/database.exception.test.ts

```diff
@@ -4,6 +4,7 @@ import { ConnectionError, MSSQLError, RequestError } from 'mssql';
 import { createMockApplicationContext } from '../../testing/testing-utilities';
 
 // Setting default Jest mocks for mssql
+//NOTE: using const here causes these tests to error out with 'Cannot access {var} before initialization
```

Comment:
should this say "using var here..."?

Reply:
const would be correct in this context. using var is what solves the issue we are seeing. I updated the second reference to also be const to make more sense

Reply:
No. It's actually not just `const`, but also `let` that causes problems. `let` [has a block scope](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)—as does `const`—where `var` [has a function or global scope](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var). It seems that with jest, using block-scoped variables outside of the outermost describe does not allow for accessing them.